systems/subscriber/sim-manager/pkg/db/sim_repo.go

```diff
@@ -124,3 +125,24 @@ func (s *simRepo) Delete(simID uuid.UUID, nestedFunc func(uuid.UUID, *gorm.DB) e
 
 	return err
 }
+func (s *simRepo) GetSimCounts() (simsCount, activeCount, deactiveCount, terminatedCount int64, err error) {
+	db := s.Db.GetGormDb()
+
+	if err := db.Model(&Sim{}).Count(&simsCount).Error; err != nil {
+		return 0, 0, 0, 0, err
+	}
+
+	if err := db.Model(&Sim{}).Where("status = ?", 1).Count(&activeCount).Error; err != nil {
```

Comment:
Can we use sim status constants rather than 1,2 or 3 ?

Reply:
Yes, I can reuse the constant declared already on db model.